### DISCUSSION TITLE
How to filter out LAN / private addresses resolution?

### DISCUSSION THREAD

**Discussion by macruspareto** [DNSCrypt/dnscrypt-proxy#1706] [Q&A] [answered]
.

**Comment by jedisct1** [comment, marked as answer]
`SYNTH` means that the response was locally synthesized. `PTR` queries for local addresses are never sent to upstream resolvers; there's nothing to configure, `dnscrypt-proxy` does it out of the box.

`224.0.*` is not a set of internal addresses. With the exception of `224.0.0.*`, everything here is routable to the Internet.

`224.0.1.1` for example, has a `PTR` record:

```sh
 drill PTR 1.1.0.224.in-addr.arpa.
```

```text
;; ANSWER SECTION:
1.1.0.224.in-addr.arpa.	28633	IN	PTR	ntp.mcast.net.
```

If you really want to block something, you can add it to your block list, like any zone.

**Reply by flesler** [reply]
I have the opposite question, is it possible to have PTR queries be properly resolved by letting it go upstream?
